### PR TITLE
TL translations - 1,269 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
Hello I have here another set of translations waiting to be reviewed. Please feel free to do so. Thank you.

### CODE REVIEW

File: book/09-git-and-other-scms/sections/import-svn.asc [hunk 1]
```diff
@@ -67,63 +67,63 @@ Date:   Sun May 3 00:12:22 2009 +0000
     fixed install - go to trunk
 ----
 
-Not only does the Author field look a lot better, but the `git-svn-id` is no longer there, either.
+Hindi lang nagmumukhang mas maayos ang Author na field, subalit ang `git-svn-id` ay wala na rin doon.
```
Comment (heywens) [inline book/09-git-and-other-scms/sections/import-svn.asc]: Author - could be translated to `May-akda`

File: book/09-git-and-other-scms/sections/import-svn.asc [hunk 2]
```diff
@@ -67,63 +67,63 @@ Date:   Sun May 3 00:12:22 2009 +0000
...
 
-You should also do a bit of post-import cleanup.
-For one thing, you should clean up the weird references that `git svn` set up.
-First you'll move the tags so they're actual tags rather than strange remote branches, and then you'll move the rest of the branches so they're local.
+Kailangan mo ring gumawa ng kakaunti paglilinis sa post-import.
```
Comment (heywens) [inline book/09-git-and-other-scms/sections/import-svn.asc]: `kakaunti paglilinis` to `kakaunting paglilinis`

File: book/09-git-and-other-scms/sections/import-svn.asc [hunk 3]
```diff
@@ -67,63 +67,63 @@ Date:   Sun May 3 00:12:22 2009 +0000
...
+Kaya, ang `git svn` ay simpleng nagdaragdag ng numero ng svn na bersyon sa pangalan ng branch sa parehong paraan lamang katulad ng pagsulat mo nito sa svn upang isalaysay ang peg-revision sa branch na iyon.
+Kung wala ka nang pakialam tungkol sa mga peg-revision, simpleng burahin ang mga iyon:
 
 [source,console]
 ----
 $ for p in $(git for-each-ref --format='%(refname:short)' | grep @); do git branch -D $p; done
 ----
 
-Now all the old branches are real Git branches and all the old tags are real Git tags.
+Ngayon lahat ng matandang mga branch ay tunay na Git na mga branch at lahat ng matandang mga tag ay tunay na Git na mga tag.
```
Comment (heywens) [inline book/09-git-and-other-scms/sections/import-svn.asc]: `matandang mga branch` - could be `mga lumang branch`.
the same goes with `matandang mga tag`

**Review by heywens — COMMENTED** [review]
Hey @josteinfrancois, I have some minor suggestion. Kindly check.

File: book/09-git-and-other-scms/sections/import-svn.asc (hunk 3 above)
Comment (josteinfrancois) [reply book/09-git-and-other-scms/sections/import-svn.asc]: haha this made me laugh sorry. changing it now

**Comment by josteinfrancois** [comment]
Thank you for pointing those out @heywens . I have updated the file based on all your comments.

**Pull request merged**